src/main/java/com/google/sps/data/DatastoreScheduledInterviewDao.java

```diff
@@ -156,13 +157,19 @@ public Entity scheduledInterviewToEntityForUpdate(ScheduledInterview scheduledIn
     return scheduledInterviewEntity;
   }
 
-  private List<Entity> getEntitiesInRange(long minTime, long maxTime, Optional<Filter> filterOpt) {
-    // Datastore only supports one inequality filter per query. To work around this, the max filter
-    // ensures that the start time is at most 60 minutes before the maxTime.
+  private List<Entity> getEntitiesInRange(
+      Instant minTime, Instant maxTime, Optional<Filter> filterOpt) {
     Filter minFilter =
-        new FilterPredicate("startTime", FilterOperator.GREATER_THAN_OR_EQUAL, minTime);
+        new FilterPredicate(
+            "startTime", FilterOperator.GREATER_THAN_OR_EQUAL, minTime.toEpochMilli());
+    // Queries can only perform inequality filters on one parameter, and so instead
+    // of using endTime for the maxFilter, startTime is used and the maxTime has 60
+    // minutes subtracted from it to be equal to the latest possible startTime.
     Filter maxFilter =
-        new FilterPredicate("startTime", FilterOperator.LESS_THAN_OR_EQUAL, (maxTime - 3600000));
+        new FilterPredicate(
+            "startTime",
+            FilterOperator.LESS_THAN_OR_EQUAL,
+            maxTime.minus(60, ChronoUnit.MINUTES).toEpochMilli());
```

Comment:
Add a comment to this function that states `maxTime` needs to be the first time after the desired range and with an example. Like if we wanted interviews starting >= 2:00PM and ending <= 6:00PM on 6/6/2020 then `maxTime` needs to be 6:00PM on that day.